### ISSUE TITLE
Newman does not run folders in order

### ISSUE THREAD

**Issue by slackboxster** [postmanlabs/newman#359]
I have a set of folders named with numbers so that they sort alphabetically:
```
010 - General
020 - MaintenanceJobs
030 - BigMaintenanceJobs
```
At the very least, my 010 General *MUST* run before the other two, as General contains a login request that populates an authentication token environment variable.

In my specific case, the file is running the 010 General last. It does not appear that newman respects the names of the folders. At the very least, a flag to enable alphabetical ordering of tests would be helfpul.

That and I have the tests physically ordered in postman in the order above, so Postman is not exporting the collection with ordering respected.

**Comment by slackboxster** [comment]
So this is probably a bug with the export from postman more than it is with the newman runner, but I notice the collection export json file looks like this:

```
...
"folders": [
		{
			"id": "009a8e18-9e7a-9b83-cfe3-9052a86d548a",
			"name": "020 - MaintenanceJobs",
			"description": "",
			"order": [
				"2bca1ed3-fcb6-62ae-404a-f6f259ab8d6a",
				"82b193ab-596e-a9e4-7092-797966f6a31f",
				"75339156-1732-cb78-43f1-4f886f21dc0f",
				"1be742c5-9653-5c08-e4cb-ea8b08abbc01",
				"20d16dc3-84d1-5600-188a-5ef5d206fcb3",
				"4ec767ca-ea78-ac1d-22f7-bf7c38e68b82",
				"8a84e028-674b-21e5-4ad1-5521aec38135",
				"9938340d-15bd-1388-9a6b-a159d4b929e1",
				"319382c4-d5db-ea13-00ff-67f87cd6448e",
				"549971a6-509a-dd70-dc4c-ad9303c5ac67"
			],
			"owner": "64615",
			"collectionId": "2c5ad8b0-47d3-ca47-6844-9cde38137552"
		},
		{
			"id": "e4e8d7cb-7a33-af48-228e-dd3dc7b7177f",
			"name": "030 - CredentialProviderMaintenanceJobs",
			"description": "",
			"order": [
				"dafc1da7-a401-c696-8592-875e03231048",
				"17e091b0-c031-2322-997c-e4f2b63507c2",
				"c10990c4-0153-6b7e-a481-130057428f96",
				"2775149d-d451-6679-5867-427317bcad10",
				"ecc72d1b-04be-0a0d-e0e4-fcbbf8383d97",
				"693e5f3a-e3e0-4472-d3ad-552a4e486a5e",
				"70d21b7f-a99c-9098-dff9-7ac5dc445efe",
				"d8ba1e2d-80f6-b601-9de8-7bc264cd8022"
			],
			"owner": "64615",
			"collectionId": "2c5ad8b0-47d3-ca47-6844-9cde38137552"
		},
		{
			"owner": "294081",
			"lastUpdatedBy": "294081",
			"lastRevision": 168101135,
			"id": "08e19966-d045-85d9-128a-d81f5c6b336f",
			"name": "010 - General",
			"description": "",
			"order": [
				"8d1f2b78-a463-c07f-9dcb-842fd19b96f7",
				"a1cc7730-8323-0bd4-312c-2c293d25700c"
			],
			"collectionId": "2c5ad8b0-47d3-ca47-6844-9cde38137552"
		}
	],
...
```

**Comment by slackboxster** [comment]
So basically, Postman is not ordering the folders properly in the json when it exports.

**Comment by abhijitkane** [comment]
@slackboxster We fixed this a few versions ago. Can you head to chrome:/extensions, enable 'Developer Mode', and click 'Update extensions now'?

**Comment by czardoz** [comment]
Should be fixed in Postman v4.1.2. @slackboxster feel free to reopen the issue if you're still having  problems :)

**Comment by 21anujashah** [comment]
My postman version is 7.3.4. I export collection in the JSON file. I run that collection from using Newman. I found that the requests are not running in order. I have one request to set the environment variable and the next request to take the environment and run it. But Newman runs the second request first, it does not have any environment. One thing, I found that my collection has more than 8 folders and each folder has subfolders. Each folder has many requests. I found that few requests are not run by Newman. Newman just skips the few requests and does not run in order.  I do not write any code to skip request(setNextRequest)
